core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -79,15 +70,15 @@ angular.module('oppia').directive('aboutPage', [
 
             var hash = WindowRef.nativeWindow.location.hash.slice(1);
             if (hash === ctrl.TAB_ID_FOUNDATION || hash === 'license') {
```

Comment:
Add `'licence'` to some constant like the other tabs.

Reply:
Done.

Reply:
Not done?

Reply:
Now done. I forgot to put in this line.